src/widgets/reflow.rs

```diff
@@ -51,7 +51,7 @@ where
     O: Iterator<Item = (I, Alignment)>,
     I: Iterator<Item = StyledGrapheme<'a>>,
 {
-    pub fn new(lines: O, max_line_width: u16, trim: bool) -> Self {
+    pub const fn new(lines: O, max_line_width: u16, trim: bool) -> Self {
```

Comment:
Consider ignoring the `clippy::missing-const-for-fn` lint at the top level, as once a function is const, a changes future to the _implementation_ of a function can impact the ability for it to be const, and that is _technically_ a semver break.

Reply:
This module is private, so I'm fine with this change.